packages/mermaid/src/docs/syntax/flowchart.md

```diff
@@ -542,6 +542,56 @@ flowchart TD
     A@{ shape: tag-rect, label: "Tagged process" }
 ```
 
+## Special shapes in Mermaid Flowcharts (v11.3.0+)
+
+Mermaid also introduces 2 special shapes to enhance your flowcharts: **icon** and **image**. These shapes allow you to include icons and images directly within your flowcharts, providing more visual context and clarity.
+
+### Icon Shape
+
+You can use the `icon` shape to include an icon in your flowchart. To use icons, you need to register the icon pack first. Follow the instructions provided [here](../config/icons.md). The syntax for defining an icon shape is as follows:
+
+```mermaid-example
+flowchart TD
+    A@{ icon: "fa:user", form: "square", label: "User Icon", pos: "t", h: 60 }
```

Comment:
Could we switch to an icon pack that is available on our site, e.g.

```suggestion
    A@{ icon: "logos:zod", form: "square", label: "User Icon", pos: "t", h: 60 }
```

![image](https://github.com/user-attachments/assets/825b0665-09c2-43ad-b193-b1cbb6fc7b9e)